extensions/stac-eo-spec.md

```diff
@@ -82,13 +84,104 @@ numbers of several popular instruments.
 | lwir11      | 10.5 - 11.5     |           |           | 10        |            | 31    |
 | lwir12      | 11.5 - 12.5     |           |           | 11        |            | 32    |
 
+
+#### Associating assets with bands
```

Comment:
I added that in #245 as well. I'd suggest mixing both approaches, I have the format description, you have the more human-readable description.

So, I'd suggest changing this a little to:
```
## Associating assets with bands

Asset definitions that contain band data should reference the band index. Each asset should provide a "eo:bands" property that is an array of 0 based indexes to the correct Band Objects.

### Item `Asset Object` fields
| Field Name | Type     | Description                                  |
| ---------- | -------- | -------------------------------------------- |
| eo:bands   | [number] | Lists the band names available in the asset. |

See [landsat8-merged.json](examples/landsat8-merged.json) for a full example.
...
```

What I also didn't thought of yesterday, that we wanted to add additional values to this field (for example offset and scale, see #245). So this would change the structure of this field a bit, but shouldn't be a big problem.